ontopy/excelparser.py

```diff
@@ -29,118 +35,248 @@ def create_ontology_from_excel(  # pylint: disable=too-many-arguments
     excelpath: str,
     concept_sheet_name: str = "Concepts",
     metadata_sheet_name: str = "Metadata",
+    imports_sheet_name: str = "ImportedOntologies",
     base_iri: str = "http://emmo.info/emmo/domain/onto#",
     base_iri_from_metadata: bool = True,
+    imports: list = None,
     catalog: dict = None,
+    force: bool = False,
 ) -> Tuple[ontopy.ontology.Ontology, dict]:
     """
     Creates an ontology from an excelfile.
 
     Catalog is dict of imported ontologies with key name and value path.
     """
+    # Get imported ontologies from optional "Imports" sheet
+    if not imports:
+        imports = []
+    try:
+        imports_frame = pd.read_excel(
+            excelpath, sheet_name=imports_sheet_name, skiprows=[1]
+        )
+    except ValueError:
+        pass
+    else:
+        imports.extend(imports_frame["Imported ontologies"].to_list())
+
     # Read datafile TODO: Some magic to identify the header row
     conceptdata = pd.read_excel(
         excelpath, sheet_name=concept_sheet_name, skiprows=[0, 2]
     )
     metadata = pd.read_excel(excelpath, sheet_name=metadata_sheet_name)
     return create_ontology_from_pandas(
-        conceptdata, metadata, base_iri, base_iri_from_metadata, catalog
+        data=conceptdata,
+        metadata=metadata,
+        imports=imports,
+        base_iri=base_iri,
+        base_iri_from_metadata=base_iri_from_metadata,
+        catalog=catalog,
+        force=force,
     )
 
 
-def create_ontology_from_pandas(  # pylint: disable=too-many-locals,too-many-branches,too-many-statements
+def create_ontology_from_pandas(  # pylint:disable=too-many-locals,too-many-branches,too-many-statements,too-many-arguments
     data: pd.DataFrame,
     metadata: pd.DataFrame,
+    imports: list,
     base_iri: str = "http://emmo.info/emmo/domain/onto#",
     base_iri_from_metadata: bool = True,
     catalog: dict = None,
+    force: bool = False,
 ) -> Tuple[ontopy.ontology.Ontology, dict]:
     """
     Create an ontology from a pandas DataFrame.
     """
 
-    # Remove Concepts without prefLabel and make all to string
+    # Remove Concepts without prefLabel and make data to string
```

Comment:
I don't fully understand this comment - something about the structure at the end that doesn't quite make sense.
Also, I don't see that `data` ends up being a `str`?